src/api.ts

```diff
@@ -35,20 +35,56 @@ export class LaunchDarklyAPI {
 		return JSON.parse(data);
 	}
 
-	async getFeatureFlag(projectKey: string, flagKey: string, envKey?: string): Promise<Flag> {
+	async getFeatureFlag(projectKey: string, flagKey: string, envKey?: string): Promise<FeatureFlag> {
 		const envParam = envKey ? '?env=' + envKey : '';
 		const options = this.createOptions(`flags/${projectKey}/${flagKey + envParam}`);
 		const data = await rp(options);
-		return new Flag(JSON.parse(data));
+		return new FeatureFlag(JSON.parse(data));
 	}
 
-	private createOptions(path: string) {
-		return {
+	async patchFeatureFlag(projectKey: string, flagKey: string, value?: PatchComment): Promise<FeatureFlag> {
+		const options = this.createOptions(`flags/${projectKey}/${flagKey}`, 'PATCH', value);
+		const data = await rp(options);
+		return new FeatureFlag(JSON.parse(data));
+	}
+
+	async patchFeatureFlagOn(projectKey: string, flagKey: string, enabled: Boolean, envKey?: string) {
+		let patch = new PatchOperation();
+		patch.path = `/environments/${this.config.env}/on`;
+		patch.op = 'replace';
+		patch.value = enabled;
+		let patchOp = new PatchComment();
+		patchOp.comment = 'VS Code Updated';
+		patchOp.patch = [patch];
+		return this.patchFeatureFlag(projectKey, flagKey, patchOp);
+	}
+
+	async getFeatureFlags(projectKey: string, envKey?: string): Promise<Array<FeatureFlag>> {
+		const envParam = envKey ? '?env=' + envKey : '';
+		const options = this.createOptions(`flags/${projectKey}/${envParam}&summary=false`);
+		const data = await rp(options);
+		const flags = JSON.parse(data).items;
+		flags.forEach((flag: FeatureFlag) => {
+			return flag;
+		});
```

Comment:
Does this loop do anything?

Reply:
it may be an artifact left over from when there was a difference between `Flag` and `FeatureFlag` I'll double check